src/DependencyInjection/GraphQLiteCompilerPass.php

```diff
@@ -271,14 +271,11 @@ public function process(ContainerBuilder $container): void
             }
         }
 
-        if (!empty($customTypes)) {
-            $definition = $container->getDefinition(StaticTypeMapper::class);
-            $definition->addMethodCall('setTypes', [$customTypes]);
-        }
-        if (!empty($customNotMappedTypes)) {
-            $definition = $container->getDefinition(StaticTypeMapper::class);
-            $definition->addMethodCall('setNotMappedTypes', [$customNotMappedTypes]);
-        }
+        $staticMapperDefinition = new Definition(StaticTypeMapper::class, [
+            '$types' => $customTypes,
+            '$notMappedTypes' => $customNotMappedTypes
+        ]);
+        $container->setDefinition('graphql.type_mapper', $staticMapperDefinition);
```

Comment:
```suggestion
        $container->setDefinition(StaticTypeMapper::class, $staticMapperDefinition);
```

Let's preserve the previous DI service id